framework/PageForm/Inputs/PageFormCreatableSelect.tsx

```diff
@@ -55,12 +55,17 @@ export function PageFormCreatableSelect<
             const values: { name: string }[] = getValues(props.name);
             onChange(!chip ? [] : values.filter((v: { name: string }) => v.name !== chip));
           }}
-          onHandleSelection={(v) => {
-            const values: string[] = getValues(props.name);
-            if (values) {
-              return onChange([...values, v]);
+          onHandleSelection={(item) => {
+            let newValue;
+            const selectedItem = item as { name: string };
+            const values: { name: string }[] = getValues(props.name);
+
+            if (values.find((value) => value.name === selectedItem.name)) {
```

Comment:
Added deduping logic

Reply:
@marshmalien Good fix Marli. I was about to write a ticket for this, you got this covered 🚀

Reply:
there is one more on skip tags
![image](https://github.com/ansible/ansible-ui/assets/95239214/7e735cb9-0e34-4990-8cf4-c0ec4e624871)